deployments/ansible/roles/collector/defaults/main.yml

```diff
@@ -34,3 +34,6 @@ splunk_fluentd_config_source: ""
 
 agent_bundle_dir: /usr/lib/splunk-otel-collector/agent-bundle
 splunk_collectd_dir: /usr/lib/splunk-otel-collector/agent-bundle/run/collectd
+
+# Configure otel collector service to use a proxy
+splunk_service_proxy: ""
```

Comment:
What do you think about `splunk_otel_collector_proxy` name?

Reply:
yes it is more accurate